airflow/models.py

```diff
@@ -981,7 +981,8 @@ def run(
             if self.try_number == 0:
                 msg = "First run"
             else:
-                msg = "Attempt {self.try_number} out of {tot_tries}"
+                msg = "Attempt {} out of {}".format(self.try_number,
```

Comment:
I think this will now say "Attempt 1" on the 2nd try (1st retry) since the string is formatted before the counter is incremented. Maybe we need a message that's more clear since it looks like the current one threw you off...